az-core/src/main/java/azkaban/Constants.java

```diff
@@ -65,6 +68,9 @@
 
   public static class ConfigurationKeys {
 
+    // Configures Azkaban Flow Version
+    public static final String AZKABAN_FLOW_VERSION = "Azkaban-Flow-Version";
```

Comment:
There is no restrict naming convention for the YAML file. But I found "key-name", "key_name" and "key name" are valid formats from below links:
https://learnxinyminutes.com/docs/yaml/
http://ess.khhq.net/wiki/YAML_Tutorial
https://bitbucket.org/asomov/snakeyaml/wiki/Documentation#markdown-header-yaml-syntax
Also I think "azkaban.flow.version" might be a little confusing since users might think it means the actual version of a flow file.